### PR TITLE
Lc284

### PR DESCRIPTION
**Pull request by snowan**

### CODE REVIEW

**Comment by snowan** [comment]
@sunboman easy~~~ bearrrrr

File: 2021/python/284.peeking-interator.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+class PeekingIterator:
+
+    def __init__(self, iterator):
+        """
+        Initialize your data structure here.
+        :type iterator: Iterator
+        """
+        self.iter = iterator
+        self.tmp = self.iter.next() if self.iter.hasNext() else None
```
Comment (sunboman) [inline 2021/python/284.peeking-interator.py:45]: `self.top = self.next()`

Comment (snowan) [reply 2021/python/284.peeking-interator.py:45]: Nooooo,, this is init step, cannot get next()

Comment (sunboman) [reply 2021/python/284.peeking-interator.py:45]: why?????????????????????????????????????????????

**Review by snowan — COMMENTED** [review]
Don’t know, it just throws exception : not init

**Pull request merged**